src/main/java/itc/converters/ElastixSimilarity2DToAffineTransform2D.java

```diff
@@ -63,6 +64,48 @@
 	 */
 	public static AffineTransform2D convert(ElastixSimilarityTransform2D elastixSimilarityTransform2D )
 	{
-		throw new UnsupportedOperationException("PR appreciated!");
+		// fetch values from elastix transform
+		//
+		final double angle = elastixSimilarityTransform2D.getRotationAnglesInRadians()[2]; // Gets Z axis rotation
+		final double scalingFactor = elastixSimilarityTransform2D.getScalingFactor();
+		final double[] rotationCenterInMillimeters = elastixSimilarityTransform2D.getRotationCenterInMillimeters();
+		final double[] translationInMillimeters = elastixSimilarityTransform2D.getTranslationInMillimeters();
+
+		// convert
+		//
+		final double[] rotationCentrePositive = new double[ 2 ];
```

Comment:
just a detail : rotationCenterPositive instead of rotationCentrePositive

Reply:
Most of this code is based on ```ElastixEuler2DToAffineTransform2D```, so it would need to be changed there too: https://github.com/image-transform-converters/image-transform-converters/blob/master/src/main/java/itc/converters/ElastixEuler2DToAffineTransform2D.java#L62 I can change the UK spelling to the American spelling in both files if you'd like?

Reply:
Fine with me!

Reply:
This is fixed now. I noticed it's the same for a number of the converters, so I changed it there too.